### PR TITLE
[exporter/tanzuobservability] Refactor histogram codes

### PR DESCRIPTION
**Pull request by keep94**
Make histogram code be more like the WF proxy code.

**Description:** 
Refactor the histogram code to make it more like the WF proxy code. Also make exponential delta histogram code more accurate.

**Testing:**
Unit testing.

### CODE REVIEW

File: exporter/tanzuobservabilityexporter/metrics.go [hunk 1]
```diff
@@ -51,10 +51,10 @@ var (
 
 var (
 	// Specifies regular histogram
```
Comment (oppegard) [inline exporter/tanzuobservabilityexporter/metrics.go:53]: This comment and the one below seem redundant given the name of the vars and structs.

Comment (keep94) [reply exporter/tanzuobservabilityexporter/metrics.go]: Done. Thanks for the feedback.

**Review by thepeterstone — APPROVED** [review]

**Review by oppegard — APPROVED** [review]

**Comment by dmitryax** [comment]
@open-telemetry/collector-approvers anyone else review this please?

**Review by jpkrohling — COMMENTED** [review]
Other than the changelog comment, this looks good to me from a structural level. The component's code has been validated by code owners.

**Comment by keep94** [comment]
Thanks @jpkrohling I added a .yaml file under the unreleased directory for CHANGELOG.md.

File: unreleased/29269_and_29262.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+change_type: enhancement
+component: tanzuobservabilityexporter
+note: "Improve alorithm to translate OTEL delta exponential histograms into tanzu observability histograms"
```
Comment (jpkrohling) [inline unreleased/29269_and_29262.yaml:3]:
```suggestion
note: "Improve algorithm to translate OTEL delta exponential histograms into tanzu observability histograms"
```

**Comment by jpkrohling** [comment]
> Could it be a flaky test?

Yes

File: unreleased/29269_and_29262.yaml (hunk 2 above)
Comment (keep94) [reply unreleased/29269_and_29262.yaml]: Done. alorithm -> algorithm.

**Comment by keep94** [comment]
Hi jpkrohling, I fixed the misspelling that you caught in my CHANGELOG entry.  Good catch by the way. Please reach out if there is anything else. Thanks.

**Review by jpkrohling — APPROVED** [review]

**Pull request merged**